<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HybriddekningDialogBase</class>
 <widget class="QDialog" name="HybriddekningDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hybriddekning</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>410</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>691</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>561</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rasterfile for saving results (recommended)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtDem">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>541</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseTxt">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpMode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Mode</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>381</width>
      <height>17</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QRadioButton" name="rb_calcSignal">
     <property name="text">
      <string>Calculate signal</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_optimize">
     <property name="text">
      <string>Optimize</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_plotHeight">
     <property name="text">
      <string>Plot height profile </string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpLayers">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>691</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Layers</string>
   </property>
   <widget class="QComboBox" name="comSurface">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>531</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Surface layer:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comTerrain">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>531</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Terrain layer:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comAntenna">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>531</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Antenna layer:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comRoad">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>531</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Road layer:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>HybriddekningDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>HybriddekningDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
